website/docs/d/cloudtrail_service_account.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "aws"
+page_title: "AWS: aws_cloudtrail_service_account"
+sidebar_current: "docs-aws-datasource-cloudtrail-service-account"
+description: |-
+  Get AWS CloudTrail Service Account ID for storing trail data in S3.
+---
+
+# aws\_cloudtrail\_service\_account
+
+Use this data source to get the Account ID of the [AWS CloudTrail Service Account](http://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-supported-regions.html)
+in a given region for the purpose of allowing CloudTrail to store trail data in S3.
+
+## Example Usage
+
+```hcl
+data "aws_cloudtrail_service_account" "main" {}
+
+resource "aws_s3_bucket" "bucket" {
+  bucket        = "tf-cloudtrail-logging-test-bucket"
+  force_destroy = true
+
+  policy = <<EOF
+{
+	"Version": "2008-10-17",
+	"Statement": [
+		{
+        			"Sid": "Put bucket policy needed for trails",
```

Comment:
Can you fix indentations there? 2 spaces would be really nice :)

Reply:
Will do.